.editorconfig

```diff
@@ -0,0 +1,651 @@
+root = true
```

Comment:
Please ignore. This is to keep the handling of tabulators in the bash scripts simple.